### ISSUE TITLE
Strings

### ISSUE THREAD

**Issue by tomahock** [FogosPT/fogosmobile#115]
Change:
Bombeiros -> Operacionais
ANPC -> ANEPC

**Comment by Vanethos** [comment]
Does it include this markdown text?

```
- __HUMANOS__ - Bombeiros, Força Especial de Bombeiros, PSP, Forças Armadas, INEM, Equipas Sapadores Florestais, GNR, GIPS Grupo Intervenção de Proteção e Socorro
- __TERRESTRES__ - Veículos rodoviários
- __AÉREOS__ - Helicópteros / Aviões
```

**Comment by Vanethos** [comment]
Closed via #127
